relay-general/src/store/transactions/processor.rs

```diff
@@ -396,6 +397,20 @@ impl Processor for TransactionsProcessor<'_> {
         Ok(())
     }
 
+    fn process_measurements(
+        &mut self,
+        value: &mut crate::protocol::Measurements,
+        _meta: &mut Meta,
+        _state: &ProcessingState<'_>,
+    ) -> ProcessingResult {
+        let measurements = &mut value.0;
+        let huge_number: usize = todo!();
```

Comment:
want to get max length from aggregatorconfig, since that's how it's done in `fn validate_metric_name`, not sure how this can be done, especially within a trait method like this